src/commands/goto.ts

```diff
@@ -108,12 +112,12 @@ function executeGotoLine(editor: vscode.TextEditor, count: number, extend: boole
     }
```

Comment:
This is not strictly necessary, but it's usually good practice to have trailing commas after the last element of an array / object, the reasoning being that if we add new members / items, adding an additional comma after the last item will "corrupt" `git blame` and similar commands that check diffs line-by-line.